libsecret.go

```diff
@@ -56,6 +59,28 @@ func (e *secretsError) Error() string {
 
 var errCollectionNotFound = errors.New("The collection does not exist. Please add a key first")
 
+func decodeKeyringString(src string) string {
```

Comment:
Seems like a good function to have some unit tests for ;)

Reply:
ok, i'll add unit test